script/vsts/platforms/windows.yml

```diff
@@ -58,7 +63,7 @@ jobs:
           BUILD_ARCH: $(buildArch)
           CI: true
           CI_PROVIDER: VSTS
-          NPM_BIN_PATH: "C:\\hostedtoolcache\\windows\\node\\12.13.1\\x64\\npm.cmd"
+          # NPM_BIN_PATH: "C:\\hostedtoolcache\\windows\\node\\12.13.1\\x64\\npm.cmd"
```

Comment:
After you rebased, consider that this number appears in these places:
![image](https://user-images.githubusercontent.com/16418197/87665385-73973800-c72c-11ea-9e1e-8da59fbc1f41.png)